tests/sls/homu/valid_repos.py

```diff
@@ -0,0 +1,33 @@
+import toml
+import urllib.request
+from urllib.error import URLError
+from tests.util import Failure, Success
+
+
+def getStatus(url):
+    '''
+    Consumes a url string and returns the status code of a GET request
+    '''
+    try:
+        response = urllib.request.urlopen(url).status
+    except URLError:
+        response = url
+    return response
+
+
+def run():
+    # repository configuration dictionary from homu
+    repo_cfg = toml.load('/home/servo/homu/cfg.toml')['repo']
+    VCS = "https://github.com/"
+    # extracting owner and repo from the configuration dict
+    # and formatting it to more easily form a url to submit a request to
+    homu_repos = [repo_cfg[repo_title]['owner']+'/'+repo_title
```

Comment:
I prefer to use string formatting, e.g. `'{}/{}'.format(repo['owner'], repo['name'])`